app/components/provider_interface/conditions_form_component.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= form_with model: form_object, url: provider_interface_application_choice_offer_conditions_path(application_choice), method: method do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-l"><%= application_choice.application_form.full_name %></span>
+    <%= form_heading %>
+  </h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_collection_check_boxes(:standard_conditions,
+                                         standard_conditions_checkboxes,
+                                         :id,
+                                         :name,
+                                         legend: { size: 'm' }) %>
+
+      <%= f.govuk_fieldset(
+        legend: {
+          text: t('.further_conditions'),
+          class: 'govuk-heading-m govuk-!-margin-bottom-1 app-add-another__heading',
+          tabindex: '-1',
```

Comment:
This makes it focusable, as per the prototype.

I'm also going to add an id for this element and focus it when the remove button is clicked (also as per prototype)